<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Estimate</class>
 <widget class="QWidget" name="Estimate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/Request/EstimateAndRequest.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="SourcTxt">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>&lt;Source Address&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="VipBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VIP Trip</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DestinationTxt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>271</width>
     <height>79</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>&lt;Destination 0 Address&gt;  &lt;Destination 1 Address&gt; ... &lt;Final Destination Address&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EstimateBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Address should be like this :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;GalaxyName,PlanetName&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
